### PR TITLE
[onert/test] Fix mixed test

### PR DESCRIPTION
**Pull request by hseok-oh**
Now AvgPool2D and MaxPool2D internal IR is merged to Pool2D.
So backend mapping is not working, and test uses cpu backend only.

This commit updates backend mapping on mixed test to use acl-cl and acl-neon backend.

Signed-off-by: Hyeongseok Oh <hseok82.oh@samsung.com>

### CODE REVIEW

**Review by chunseoklee — APPROVED** [review]

File: infra/scripts/test_ubuntu_runtime_mixed.sh [hunk 1]
```diff
@@ -55,8 +55,8 @@ echo "GeneratedTests.squeeze_relaxed" >> $SKIPLIST_PREFIX.union
 
 # Run the test
 export OP_BACKEND_Conv2D="cpu"
-export OP_BACKEND_MaxPool2D="acl_cl"
-export OP_BACKEND_AvgPool2D="acl_neon"
+export OP_BACKEND_Pool2D="acl_cl"
+export OP_BACKEND_FullyConnected="acl_neon"
```
Comment (glistening) [inline infra/scripts/test_ubuntu_runtime_mixed.sh:59]: @hseok-oh Is this line related to `MaxPool` and `AvgPool` changes?

Comment (hseok-oh) [reply infra/scripts/test_ubuntu_runtime_mixed.sh:59]: @glistening Right. #3853 merges internal IR `MaxPool2D` and `AvgPool2D` to `Pool2D`, and OP-backend mapping uses internal IR list. So these env variables cannot handle backend mapping.

Comment (glistening) [reply infra/scripts/test_ubuntu_runtime_mixed.sh:59]: @hseok-oh I am asking line 59 (`OP_BACKEND_FullyConnected`). ( I chosen line 59, but github highlights including 58. )

Comment (hseok-oh) [reply infra/scripts/test_ubuntu_runtime_mixed.sh:59]: `OP_BACKEND_MaxPool2D` and `OP_BACKEND_AvgPool2D` do nothing. So remain OP code on inception test model is `FullyConnected`, `Concat`, and `Softmax`. So I choosed `FullyConnected`.

**Review by glistening — APPROVED** [review]
LGTM
